packages/kbn-user-profile-components/src/user_profiles_selectable.tsx

```diff
@@ -425,16 +436,19 @@ export const UserProfilesSelectable = <Option extends UserProfileWithAvatar | nu
 };
 
 type SelectableOption = EuiSelectableOption<Partial<UserProfileWithAvatar>>;
+export type NullOptionProps = Partial<Pick<EuiSelectableOption, 'append'>>;
 
 function toSelectableOption(
   userProfile: UserProfileWithAvatar | null,
-  nullOptionLabel?: string
+  nullOptionLabel?: string,
+  nullOptionProps?: NullOptionProps
 ): SelectableOption {
   if (userProfile) {
     return {
       key: userProfile.uid,
       label: getUserDisplayName(userProfile.user),
       data: userProfile,
+      'data-test-subj': `userProfileSelectableOption-${userProfile.user.username}`,
```

Comment:
it is a bit hacky, but this is what was more convenient in tests, since I know the username, but don't know the u_id of the current user. I guess I can make an API request to fetch it. but not sure it is worth it in this case

Reply:
> but not sure it is worth it in this case

It's likely not worth it if it's not trivial. I was just not sure what happens if we have duplicated test attributes in prod - I assume nothing?

Reply:
nothing! happens relatively often